src/scalarstats.jl

```diff
@@ -390,6 +390,47 @@ function entropy{T<:Real}(p::AbstractArray{T}, b::Real)
     return entropy(p) / log(b)
 end
 
+"""
+    renyientropy(p, α)
+
+Compute the Rényi (generalised) entropy of order `α` of an array `p`.
```

Comment:
TBH I've got so confused by s's and z's now that I can't even work out which spelling is which most of the time any longer! Anyway, corrected.